schemas/Interaction.json

```diff
@@ -26,11 +26,8 @@
         {"const": "WebRoomHeaderContextMenuInviteItem", "description":  "User accessed the room invite flow using the context menu on the header of a room in Element Web/Desktop." },
         {"const": "WebRightPanelMemberListInviteButton", "description":  "User accessed the room invite flow using the button at the top of the room member list in the right panel of Element Web/Desktop." },
         {"const": "WebRightPanelRoomUserInfoInviteButton", "description":  "User invited someone to room by clicking invite on the right panel user info card in Element Web/Desktop." },
-        {"const": "MobileSendMessageButton", "description": "User tapped the send message button."},
-        {"const": "MobileSideMenuSelectedSpace", "description": "User tapped the already selected space from the space list."},
-        {"const": "MobileSideMenuSwitchSpace", "description": "User tapped an unselected space from the space list -> space switching should occur."},
-        {"const": "MobileUserNotification", "description": "User tapped an application notification"},
-        {"const": "MobileMatrixToLink", "description": "User tapped a matrix to link."}
+        {"const": "SideMenuSelectedSpace", "description": "User tapped the already selected space from the space list."},
+        {"const": "SideMenuSwitchSpace", "description": "User tapped an unselected space from the space list -> space switching should occur."}
```

Comment:
Could is be `SpacePanel...` to be more inclusive as its not really a menu on Web/Desktop